### PR TITLE
Docs: Add information about Graphite/Loki integration

### PR DESCRIPTION
**Pull request by ifrost**
**What this PR does / why we need it**:

A follow-up to #33405. There is plenty of docs in the help section in settings so this is just to mentioned the feature in the main docs.

**Which issue(s) this PR fixes**:

Fixes: N/A

**Special notes for your reviewer**: N/A

### CODE REVIEW

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Added doc change request.

File: docs/sources/datasources/graphite.md [hunk 1]
```diff
@@ -244,3 +244,9 @@ datasources:
     jsonData:
       graphiteVersion: "1.1"
 ```
+
+## Integration with Loki
+
+Graphite queries are automatically converted to Loki queries when the data source changes in Explore. Loki label names and values are extracted from Graphite queries according to mappings provided in Graphite data source configuration. Queries using tags with `seriesByTags()` are transformed automatically without any extra setup.
+
+Please check Graphite data source settings for more details.
```
[the remark this answers is not in the document]
Comment (ifrost) [reply docs/sources/datasources/graphite.md:252]: > Refer to the Graphite data source settings for more details.

There's a help section in data source configuration page in the application. I wanted to suggest checking it for more info and examples:

info

File: docs/sources/datasources/graphite.md [hunk 2]
```diff
@@ -244,3 +244,9 @@ datasources:
     jsonData:
       graphiteVersion: "1.1"
 ```
+
+## Integration with Loki
+
+Graphite queries get converted to Loki queries when the data source selection changes in Explore. Loki label names and values are extracted from the Graphite queries according to mappings information provided in Graphite data source configuration. Queries using tags with `seriesByTags()` are also transformed without any additional setup.
+
+Refer to the Graphite data source settings for more details.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/graphite.md:252]:
```suggestion
Refer to Graphite data source documentation for more details.
```

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Pull request merged**